site/content/docs/manifest/lb-web-service.en.md

```diff
@@ -259,6 +259,9 @@ http:
   hosted_zone: Z0873220N255IR3MTNR4
 # Also see http.alias array of maps example, above.
 ```
+<span class="parent-field">http.</span><a id="http-disable-redirect" href="#http-disable-redirect" class="field">`disable_redirect`</a> <span class="type">Boolean</span>  
```

Comment:
@dannyrandall @iamhopaul123 

Any reason why we preferred a double negation? Typically, we should avoid them as a code smell: https://refactoring.com/catalog/removeDoubleNegative.html

What do y'all think of changing the field to `http.redirect: false` and by default it's `true`.